specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/customApisDelete.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "apiName": "testCustomApi",
+    "api-version": "2016-06-01",
+    "customApi": {
+      "tags": {
+        "sampleTagKey": "sampleTagValue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {}
```

Comment:
What do we expect in the response body?

Reply:
it is confusing then as to why 200 is used instead of 204 as 204 is explicitly saying body is empty yet 200 is used for the same??

Reply:
When the custom API is deleted, a 200 is returned, when a nonexistent custom API is deleted (such as deleting the same one a second time) a 204 is returned. Both come back with an empty body